libraries/AP_BattMonitor/AP_BattMonitor_ADS1115.cpp

```diff
@@ -0,0 +1,60 @@
+#include <AP_HAL/AP_HAL.h>
+#include <AP_Common/AP_Common.h>
+#include <AP_Math/AP_Math.h>
+#include "AP_BattMonitor.h"
+#include <AP_ADC/AP_ADC.h>
+#include "AP_BattMonitor_ADS1115.h"
+
+#define debug 1
```

Comment:
This should default to 0 (false) and I'd rather this be in all caps. Let's also make this a more specific definition - e.g. BATTMON_ADS1115_DEBUG.